source/core/ut_utils.pks

```diff
@@ -473,5 +477,28 @@ create or replace package ut_utils authid definer is
   */    
   function interval_to_text(a_interval yminterval_unconstrained) return varchar2;
   
+  /*
+  * Return table of tokens character by character
+  */   
+  function tokenize_tags_string(a_tags in varchar2) return ut_varchar2_list;
+
+  /*
+  * Function that uses Dijkstra algorithm to parse mathematical and logical expression
+  * and return a list of elements in Reverse Polish Notation ( postfix )
+  * As part of execution it will validate expression.
+  */
+  function shunt_logical_expression(a_tags in varchar2) return ut_varchar2_list;
+
+  /*
+  * Function that converts postfix notation into infix
+  */
+  function convert_postfix_to_infix(a_postfix_exp in ut_varchar2_list) return varchar2;
```

Comment:
Yes, this is a proper function that returns a normal postfix notation. Thought I will include it in case we want to use in in future but we are using one that constructs SQL with member of .
I can remove it.